e2e/integration/feedbacktests.js

```diff
@@ -0,0 +1,28 @@
+/// <reference types="cypress" />
+
+context( 'Test feedback form', () => {
+
+    before( () => {
+        cy.visit( 'http://stage.one.redhat.com/' );
+        Cypress.config( 'includeShadowDom', true );
+        cy.get( '#username', { timeout: 5000 } ).type( 'one-portal-test' );
+        cy.get( '#password' ).type( 'iequeivaequ:e2OobahM3vahbaePh8' );
+        cy.get( '#submit' ).click();
+        //cy.contains( 'Feedback', { timeout: 5000 } ).click();
+
+    } );
+    Cypress.on( 'uncaught:exception', ( err, runnable ) => {
+        // returning false here prevents Cypress from
```

Comment:
Why this is required? These are supposed to fail right when something is changed?